amp/alfresco6/src/main/amp/module.properties

```diff
@@ -0,0 +1,4 @@
+module.id=$project.name
```

Comment:
If I'm not mistaking, this will become `alfresco5` and `alfresco6`? If that is true, it's no good module id.